Background.md

```diff
@@ -37,24 +39,68 @@ Een WebID duidt op meerdere 'dingen'. Het is:
+
+<div align="center">
+  <img src="images/VC.png" alt="The rollen and informatieflows die de basis vormen voor de Verifiable Credentials specificatie." width="600">
+  <p><em>Schematische weergave. Bron <a href="https://www.w3.org/TR/vc-data-model/#ref-for-dfn-verifiable-data-registries-1">Verifiable Credentials Data Model v1.1</a></em></p>
+</div>
+
+Verschillende VC-ecosystemen zullen verschillende VDR's hebben, waarbij de onderliggende technologie kan variëren. Mogelijke implementaties van een VDR omvatten vertrouwde databases, gedecentraliseerde databases, overheids-ID-databases en gedistribueerde ledgers. Het is ook mogelijk dat binnen één ecosysteem meerdere VDR's worden gebruikt om verschillende aspecten van de VC-uitwisseling te ondersteunen. De keuze voor een specifieke VDR hangt af van de context en de vereisten van het systeem. Het belangrijkste is dat de VDR zorgt voor een betrouwbaar en veilig beheer van VC-gerelateerde informatie en bijdraagt aan het vertrouwen tussen de betrokken partijen.
+
+Een Verifiable Data Registry wordt vaak gebruikt bij de intrekking (revocation) van VCs. Intrekking verwijst naar het vroegtijdig beëindigen van de geldigheid van een verifieerbare referentie. Door het gebruik van een Verifiable Data Registry kan informatie over ingetrokken VCs worden bijgehouden. Bij het verifiëren van een VC kan de verifiërende partij het register raadplegen om te controleren of de VC nog steeds geldig is.
+
+
+### Data Minimalisatie
+
+Data minimalisatie is een essentieel principe binnen Verifiable Credentials (VCs) dat tot doel heeft de hoeveelheid persoonlijke gegevens te minimaliseren die worden verzameld, verwerkt en opgeslagen. Het minimaliseren van gegevens heeft als doel de privacy van gebruikers te beschermen en het risico op ongeoorloofde toegang tot gevoelige informatie te verminderen. Binnen VCs worden alleen de noodzakelijke gegevens opgenomen om een specifieke claim te ondersteunen, waardoor overbodige persoonlijke gegevens worden geëlimineerd. Dit draagt bij aan het beperken van potentiële gegevensinbreuken en het minimaliseren van de blootstelling van persoonlijke informatie, waardoor de privacy van individuen beter wordt beschermd. Door bewust om te gaan met gegevensverzameling en -gebruik kunnen we de voordelen van Verifiable Credentials realiseren terwijl we de privacy en veiligheid waarborgen.
+
+Verschillende vormen van disclosure-methoden, manieren om informatie selectief te onthullen zonder volledige details prijs te geven, worden gebruikt binnen Verifiable Credentials om informatie te delen. Hier zijn drie belangrijke vormen van disclosure:
+
+1. **Full Disclosure**: Het delen van gegevens is momenteel een alles-of-niets proces, zowel online als offline. Veel digitale identiteitssystemen onthullen alle attributen in een digitale credential, waardoor het risico op impersonatie ontstaat.
```

Comment:
tussen haakjes toevoegen '(dus geen data minimalisatie)'

Reply:
Strict genomen kan je dataminimalisatie hebben mét full disclosure van je data. Maar dat zit 'm dan in het proof deel. Onderaan heb de uitleg van de spec toegevoegd voor de precieze details, mocht je dat interessant vinden. Voor onze documentatie is het denk ik teveel, maar ik zal wel een poging doen om het beter uit te leggen 😄

> Along with a full disclosure of all the attributes in a [verifiable credential](https://w3c.github.io/vc-imp-guide/#dfn-verifiable-credentials), standard [verifiable presentations](https://w3c.github.io/vc-imp-guide/#dfn-verifiable-presentations) reveal the actual signature. With both the data and signature in hand, a [verifier](https://w3c.github.io/vc-imp-guide/#dfn-verifier) has a complete copy of the credential. Without care, this could enable the [verifier](https://w3c.github.io/vc-imp-guide/#dfn-verifier) to impersonate the [holder](https://w3c.github.io/vc-imp-guide/#dfn-holders). Also, since the signature is the same every time this credential is presented, the signature itself is a unique identifier and becomes PII (personally identifiable information).
> 
> It is also possible to fully disclose the attributes in a zero-knowledge [verifiable credential](https://w3c.github.io/vc-imp-guide/#dfn-verifiable-credentials). Unlike non-ZKP methods, zero-knowledge methods do not reveal the actual signature; instead, they only reveal a cryptographic proof of a valid signature. Only the [holder](https://w3c.github.io/vc-imp-guide/#dfn-holders) of the signature has the information needed to present the [credential](https://w3c.github.io/vc-imp-guide/#dfn-credential) to a [verifier](https://w3c.github.io/vc-imp-guide/#dfn-verifier). This means that zero-knowledge methods provide a [holder](https://w3c.github.io/vc-imp-guide/#dfn-holders) additional protection from impersonation. Because the signature is not revealed, it also cannot be used as a unique identifier.

_https://w3c.github.io/vc-imp-guide/#zero-knowledge-proofs_

Reply:
Aha. Ja, dat begrijp ik nu. Full disclosure van de claims ... maar niet de proof. Check. Thnx!